src/pages/resultsView/enrichments/GeneBarPlot.tsx

```diff
@@ -155,11 +154,25 @@ export default class GeneBarPlot extends React.Component<IGeneBarPlotProps, {}>
         </div>)
     }
 
+    @computed private get selectedOption() {
+        if (this._label && this._geneQuery !== undefined) {
```

Comment:
Just want to confirm that `this._geneQuery` can be an empty string like `""`? Should we return the default result for that case?

Reply:
if its empty then it would be in sync mode